frontend/packages/core-api-client/src/lib/types/client/Record.ts

```diff
@@ -1,12 +1,16 @@
-import { Record, RecordList } from '../model/Record';
+import { Record, RecordList } from '../model';
 
 import { DataOperation, PartialObjectWrapper, Response } from './utils';
 
 export type RecordCreateRequest = PartialObjectWrapper<Record>;
 export type RecordCreateResponse = Response<RecordCreateRequest, Record>;
 
+export type FormLookup = { databaseId: string; formId: string };
```

Comment:
This should be in `types/client/Form.ts`

Reply:
depends on what the structure is here. the name suggests form, yes, but it's going to be used in the record client.